docs/sources/introduction/profiling-types/_index.md

```diff
@@ -20,34 +18,24 @@ Profiling is an essential tool for understanding and optimizing application perf
 
 In Pyroscope, profiling types refer to different dimensions of application performance analysis, focusing on specific aspects like CPU usage, memory allocation, or thread synchronization.
 
-For information on auto-instrumentation and supported language SDKs, refer to [Configure the client]({{< relref "../../configure-client" >}}).
-
-### Available profiling types
-
-Various languages support different profiling types.
-Pyroscope supports the following profiling types:
-
-| Profile Type       | Go    | Java  | .NET       | Ruby  | Python | Rust  | Node.js | eBPF (Go) | eBPF (Python)|
-|--------------------|-------|-------|------------|-------|--------|-------|---------|-----------|--------------|
-| CPU                | Yes   | Yes   | Yes        | Yes   | Yes    | Yes   | Yes     | Yes       | Yes          |
-| Alloc Objects      | Yes   | Yes   | Yes        |       |        |       |         |           |              |
-| Alloc Space        | Yes   | Yes   | Yes        |       |        |       |         |           |              |
-| Inuse Objects      | Yes   |       | Yes (7.0+) |       |        |       |         |           |              |
-| Inuse Space        | Yes   |       | Yes (7.0+) |       |        |       |         |           |              |
-| Goroutines         | Yes   |       |            |       |        |       |         |           |              |
-| Mutex Count        | Yes   |       | Yes        |       |        |       |         |           |              |
-| Mutex Duration     | Yes   |       | Yes        |       |        |       |         |           |              |
-| Block Count        | Yes   |       |            |       |        |       |         |           |              |
-| Block Duration     | Yes   |       |            |       |        |       |         |           |              |
-| Lock Count         |       | Yes   | Yes        |       |        |       |         |           |              |
-| Lock Duration      |       | Yes   | Yes        |       |        |       |         |           |              |
-| Exceptions         |       |       | Yes        |       |        |       |         |           |              |
-| Wall               |       |       | Yes        |       |        |       |         |           |              |
-| Heap               |       |       |            |       |        |       | Yes     |           |              |
+Pyroscope supports these profile types:
 
-## CPU profiling
+* CPU
+* Memory (allocation objects, allocation space)
+* In use objects and in-use space
+* Goroutines
+* Mutex count and duration
+* Block count and duration
+* Lock count and duration
+* Exceptions
+* Wall
+* Heap
```

Comment:
This list appears in `docs/sources/configure-client/profile-types.md` as well. Do we want to keep in both places? It might be tricky to keep both lists in sync.

Reply:
I almost created a shared file to use an include here. Might be a good idea, since we might need to list the profile types in other areas as well.

Reply:
I've created a shared file so we can include the profile type list any where we want. Thinking about using this in the Cloud Profiles and Explore Profiles docs as well.